packages/utils/src/lib/reports/utils.ts

```diff
@@ -237,7 +237,7 @@ export function compareIssueSeverity(
 type LoadedReportFormat<T extends Format> = T extends 'json' ? Report : string;
 
 export async function loadReport<T extends Format>(
-  options: Required<Pick<PersistConfig, 'outputDir' | 'filename'>> & {
+  options: Omit<NonNullable<Required<PersistConfig>>, 'format'> & {
```

Comment:
The `NonNullable<...>` type transformation doesn't have any effect here, the `Required<PersistConfig>` type doesn't include `null` or `undefined`.

```suggestion
  options: Omit<Required<PersistConfig>, 'format'> & {
```